thirdeye/thirdeye-pinot/src/main/resources/assets/css/thirdeye.css

```diff
@@ -91,6 +97,23 @@ body {
     margin-bottom: 0px;
     border: 0px;
 }
+
+.navbar-inverse .navbar-brand {
+  color: white;
+}
+
+.navbar-brand:before {
```

Comment:
Would this be better served as an image element since it is content that conveys semantic meaning? Don't think `background-image` is a11y friendly.